packages/gatsby-source-shopify/src/type-builders/location-type.ts

```diff
@@ -5,7 +5,7 @@ export function locationTypeBuilder(prefix: string): string {
         fulfillmentOrdersOptIn: Boolean!
         handle: String!
         inventoryManagement: Boolean!
-        productBased: Boolean!
+        productBased: Boolean! @deprecated(reason: "\`productBased\` will be removed 2024-07, non-product based fulfillment services are no longer supported")
```

Comment:
I would say "in version 2024-07"

Reply:
Addressed in https://github.com/gatsbyjs/gatsby/pull/39082/commits/2feb7371e217fcb915581ebf528b26b7ac43e12d